PythonTool/DataLoading.py

```diff
@@ -125,7 +125,10 @@ def plane_slice_plotting(window_number, axis_number, label, images):
 
     # Include the CNERG logo in the bottom left corner of the plot.
     image = Vi.CreateAnnotationObject("Image")
-    image.image = os.path.dirname(__file__) + "/cnerg.jpg"
+    if(os.getcwd() == os.path.dirname(os.path.abspath(__file__))):
```

Comment:
Does `os.path.dirname(os.path.abspath(__file__))` not always provide a valid path?

Reply:
You are right, it absolutely does! I am so sorry, that was me being paranoid.